index.js

```diff
@@ -9,19 +9,30 @@ const { marthaV3Handler } = require('./martha/martha_v3');
 const { fileSummaryV1Handler } = require('./fileSummaryV1/fileSummaryV1');
 const getSignedUrlV1 = require('./handlers/getSignedUrlV1');
 
+const addSecurityHeaders = (res) => {
+    res.set({
+        'Strict-Transport-Security': 'max-age=63072000',
```

Comment:
I see that https://hstspreload.org/ advises starting with a short age, verifying that it works, and later ramping up to two years. Was this taken into consideration?

Reply:
FYI we have a convention as well as actual certificates in place (in some places) that allow `local.broadinstitute.org` for HTTPS testing. And, alias that to `localhost` in the `hosts`.